### DISCUSSION TITLE
Blank page after setting CDN

### DISCUSSION THREAD

**Discussion by Hezekyel** [mozilla/hubs#5773] [Hubs Cloud Questions] [answered]
I turned on CDN workers and now a I get a blank page
How can I turn it off, or roll back? I can´t even access admin page, is also blank

**Comment by SV-AndreiC** [comment, marked as answer]
hello, check the link bellow for steps to disable CDN
https://github.com/mozilla/hubs/discussions/3138

**Reply by Hezekyel** [reply]
Thanks!! It worked. But couldn´t find the same strings, so I checked timestamps and delete everything around the time I set cloudflare
